roadmap.md

```diff
@@ -4,45 +4,47 @@ The CloudEvents Roadmap.
 
 *Setup*
 
-* Establish governance, contributing guidelines and initial stakeholders.
-* Define design goals for CloudEvents V.1.
-* Describe the scope of the specification.
-* Draft educational materials that provide context for reading the spec.
+1. Establish governance, contributing guidelines and initial stakeholders.
+1. Define design goals for CloudEvents V.1.
+1. Describe the scope of the specification.
+1. Draft educational materials that provide context for reading the spec.
 
 *0.1*
 
-* Draft specification that WG members agree *could* provide interoperability.
-* Document at least 3 sample events that conform to the specification.
-* Github repo is organized to be approachable to a engineers who might want to
+1. Draft specification that WG members agree *could* provide interoperability.
+1. Document at least 3 sample events that conform to the specification.
+1. Github repo is organized to be approachable to a engineers who might want to
 implement the spec.
-* Additional materials:
-  * Use-cases.
-  * Placeholder for protocol specification, folder and brief starter doc for
+1. Use-cases.
+1. Placeholder for protocol specification, folder and brief starter doc for
   HTTP.
-* Finalize logo.
-* Create and deploy a website that features a simple overview, email list and directs visitors to Github.
-* Store all website assets in the CloudEvents repository, under the governance
+1. Finalize logo.
+1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
+1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
 
 
 *0.2*
 
-* Have at least 2 implementations of the specification that can demonstrate interoperability.
-* Changes to the spec to facilitate adoption.
-* HTTP protocol specification.
-* Draft documentation and developer guide.
-* Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
+1. Have at least 2 implementations of the specification that can demonstrate interoperability.
+1. Changes to the spec to facilitate adoption.
+1. HTTP protocol specification.
+1. Draft documentation and developer guide.
+1. Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
 
 *0.3*
 
-* Interoperability demo.
-  * At least one open source implementation of sending and receiving events.
-  * Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
-  * Draft trigger specification (e.g. how events + actions are associated).
-* Draft documentation and user guides.
+1. Interoperability demo.
+
+  i. At least one open source implementation of sending and receiving events.
```

Comment:
These are kind of funky. Look at the rendered version, these appear to the left of their parent numbering.

Reply:
And the numbering is off a little too.